synapse/rest/client/v2_alpha/shared_rooms.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Half-Shot
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+
+from synapse.api.errors import Codes, SynapseError
+from synapse.http.servlet import RestServlet
+from synapse.types import UserID
+
+from ._base import client_patterns
+
+logger = logging.getLogger(__name__)
+
+
+class UserSharedRoomsServlet(RestServlet):
+    """
+    GET /uk.half-shot.msc2666/user/shared_rooms/{user_id} HTTP/1.1
+    """
+
+    PATTERNS = client_patterns(
+        "/uk.half-shot.msc2666/user/shared_rooms/(?P<user_id>[^/]*)",
```

Comment:
Sorry, just noticed this and your MSC [don't match up](https://github.com/matrix-org/matrix-doc/blob/hs/shared-rooms/proposals/2666-get-rooms-in-common.md#unstable-prefix). You'll want to put `/unstable/` before `/uk.half-shot.msc2666`. This matches the unstable endpoint definitions laid out in [MSC2324](https://github.com/matrix-org/matrix-doc/blob/master/proposals/2324-when-to-ship.md).

You can do so by adding `unstable=True` to this function call (and at that point the `# This is an unstable feature` comment is likely redundant).

Reply:
I don't need to. `unstable` [is True by default](https://github.com/matrix-org/synapse/blob/53f7b49f5b09aca3900c520a142496da31fc1ee3/synapse/rest/client/v2_alpha/_base.py#L32). I don't think I need to stick /unstable/ before the path either.

Reply:
Ah! My bad. Just a couple doc updates then and we're good to go!